benchmark_runner/common/prometheus/metrics-default.yaml

```diff
@@ -7,4 +8,12 @@
 - query: sum(container_memory_rss{namespace="benchmark-runner"} and on (node) kube_node_role{role="worker"}) by (node) > 0
   metricName: containerMemory-benchmark-runner
 
+- query: container_memory_cache{pod=~"virt-launcher.*", container="compute"}
+  metricName: VM Memory Cache
```

Comment:
I'm not certain that spaces in metric names work correctly.  Make sure you test this.

Reply:
Yes, works for me in Grafana and its more readable than VM-Memory-Cache